app/javascript/packs/header.js

```diff
@@ -1,44 +1,24 @@
+// Adds toggle to mobile menu
+
 document.addEventListener("turbolinks:load", function () {
-  // Handle menu trigger menu
-  document.getElementById("triggerMenu").addEventListener("click", triggerMenu);
+  document.getElementById("mobile-button").addEventListener("click", () => {
+    let mM = document.getElementById("mobile-menu");
```

Comment:
This is the ultimate nitpick, but I might suggest using longer variable names so it's a little easier to read.

Reply:
of course. I meant to change that...was using short names because i was having a bug i couldn't figure out...anyway fixed in commits.